pkg/jtracer/testutils/.nocover

```diff
@@ -0,0 +1 @@
+requires connection to OTLP receiver
```

Comment:
```suggestion
FIXME
```
when we write an explanation in this file it usually means that unit testing is not feasible without external dependencies. In this case this is testable, we just skipped it.